dev-itpro/administration/optimize-sql-al-Database-methods-and-performance-on-server.md

```diff
@@ -46,11 +46,11 @@ Any result set that is returned from a call to the Find methods discussed in the
   
 The following code shows how records are most efficiently retrieved. **FindSet** is the most efficient method to use because this example reads all records.  
   
-```  
-if FindSet then  
-  repeat  
-    // Insert statements to repeat.  
-  until Next = 0;  
+```AL  
+if FindSet() then  
```

Comment:
Aren't we also recommending not using implicit recs?

Reply:
Do you mean to use `Rec.`? This is true on most objects, except on (the same) table object.
For the sake of readability within the article, I'd prefer to omit `Rec`.

Reply:
I think it is generally not so good practice to use the FindSet/repeat/until in the same table and I am guessing/hoping it is also not happening that much in tables. Using Rec. before FindSet sounds as a more correct pattern to me indeed.